yb-voyager/cmd/importData.go

```diff
@@ -1010,6 +1007,50 @@ func getProgressAmount(filePath string) int64 {
 	return p
 }
 
+func getYBSessionInitScript() []string {
+	sessionVarsPath := filepath.Join("/", "etc", "yb-voyager", "ybSessionVariables.sql")
```

Comment:
`filepath.Join()` is used when some part of the path is given by runtime variables.
In this particular case all of the path components are known, so you can simply define a string constant.